webpack.config.js

```diff
@@ -4,7 +4,7 @@ const ImageMinimizerPlugin = require('image-minimizer-webpack-plugin');
 // Core bundle assets
 Encore
     .setOutputPath('core-bundle/public/')
-    .setPublicPath('/bundles/contaocore')
+    .setPublicPath(Encore.isDevServer() ? '/' : '/bundles/contaocore')
```

Comment:
Why is this necessary?

Reply:
> `Error´:  Unable to determine contentBase option for webpack's devServer configuration. The publicPath (/bundles/contaocore/) string does not exist in the outputPath (\contao\5.x\core-bundle\public), and so the "document root" cannot be determined.

Basically for HMR to work.

Reply:
I've changed the public path for HMR in 5545373b7f278478876758f023c1348c10c89582, this one is still necessary as it's for loading the assets via localhost / the dev server :)